applications/ParticleMechanicsApplication/README.md

```diff
@@ -36,7 +36,7 @@ Recommended references for implementation details of MPM in Kratos:
 - Iaconeta, I., Larese, A., Rossi, R., & Zhiming, G. (2016). Comparison of a material point method and a Galerkin meshfree method for the simulation of cohesive-frictional materials. *Materials*, 10(10), p. 1150.
 - Iaconeta, I., Larese, A., Rossi, R., Oñate, E. (2017). An implicit material point method applied to granular flows. *Procedia Engineering: Proceeding of the 1st International Conference on the Material Point Method (MPM2017)*, Vol 175, 226-232 MPM 2017, Delft, Netherlands.
 - Iaconeta, I., Larese, A., Rossi, R., & Oñate, E. (2018). A stabilized mixed implicit Material Point Method for non-linear incompressible solid mechanics. *Computational Mechanics*, 1-18.
-- Chandra, B., Larese, A., Iaconeta, I., Rossi, R., Wüchner, R. (2018). Soil-Structure Interaction Simulation of Landslides Impacting a Structure Using an Implicit Material Point Method. *Proceeding of the 2nd International Conference on The Material Point Method (MPM2019)*, Cambridge, United Kingdom.
+- Chandra, B., Larese, A., Iaconeta, I., Rossi, R., Wüchner, R. (2019). Soil-Structure Interaction Simulation of Landslides Impacting a Structure Using an Implicit Material Point Method. *Proceeding of the 2nd International Conference on The Material Point Method (MPM2019)*, Cambridge, United Kingdom.
```

Comment:
I think that one is online, right? So please put directly the link.

Reply:
@antonialarese